src/main/java/org/folio/service/orders/CompositePoLineValidationService.java

```diff
@@ -76,6 +76,66 @@ private List<Error> validatePoLineFormats(CompositePoLine compPOL) {
     return Collections.emptyList();
   }
 
+  public List<Error> validatePoLineMaterial(CompositePoLine compPOL) {
+
+    CompositePoLine.OrderFormat orderFormat = compPOL.getOrderFormat();
+
+    if (orderFormat == P_E_MIX) {
+      return P_E_MixCheck(compPOL);
+    } else if (orderFormat == ELECTRONIC_RESOURCE) {
+      return electronicResourceCheck(compPOL);
+    } else if (orderFormat == CompositePoLine.OrderFormat.PHYSICAL_RESOURCE) {
+      return physicalResourceCheck(compPOL);
+    } else if (orderFormat == CompositePoLine.OrderFormat.OTHER) {
+      return otherResourceCheck(compPOL);
+    }
+
+    return Collections.emptyList();
+  }
+
+  //to check if the P_E_Mix missing either physical or electronic resources
+  private List<Error> P_E_MixCheck (CompositePoLine compPOL ){
+    List<ErrorCodes> errors = new ArrayList<>();
+    if(compPOL.getPhysical()==null || compPOL.getEresource()==null){
+     errors.add(ErrorCodes.INVALID_PEMIX_POL);
+    }else if (getElectronicCostQuantity(compPOL)==0 || getPhysicalCostQuantity(compPOL)==0){
+      errors.add(ErrorCodes.INVALID_PEMIX_POL);
+    }
+    return convertErrorCodesToErrors(compPOL, errors);
+  }
+
+  // to check if electronic order format contains physical resources
+  private List<Error>  electronicResourceCheck (CompositePoLine compPOL ){
+    List<ErrorCodes> errors = new ArrayList<>();
+    if((compPOL.getPhysical()!=null)) {
```

Comment:
only 1 level of brackets are necessary, also formatting 
if (compPOL.getPhysical() != null) {